XCUITests/NightModeTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  NightModeTests.swift
+//  Client
+//
+//  Created by Usha Garlapati on 6/16/17.
+//  Copyright © 2017 Mozilla. All rights reserved.
+//
+
```

Comment:
The header for new test cases should be always the same.  You should use the following:

/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */